src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewBaseItemChrome.cs

```diff
@@ -1306,6 +1306,42 @@ private bool ShouldDrawDottedLinesFocusVisual()
 
 			return GetOwner() != null;
 		}
+
+		//Uno only
+		#region Uno Specific Logic
+		private Brush GetTodaySelectedBackgroundBrush()
+		{
+			var pOwner = GetOwner();
+			if (pOwner is { } && m_isToday && m_isSelected)
+			{
+				return pOwner.m_pTodaySelectedBackground;
+			}
+
+			return null;
```

Comment:
If it can returns `null` it should be `**Find**TodaySelectedBackgroundBrush` (FT rules ! ;) )